docs/Way-of-Working/Offboarding.md

```diff
@@ -0,0 +1,30 @@
+# Off Boarding
+
+We're sad to see you go! But if you do, here's what not to forget.
+
+## Webex
+
+- Leave our Webex Team.
+
+## Github
+
+- Make sure you handed any ownership you may still over to the team.
+- Make sure to leave our team  [AI Validation Team](https://github.com/orgs/MinBZK/teams/ai-validation-team).
+- If you are leaving MinBZK altogether, make sure to leave [MinBZK Org](https://github.com/orgs/MinBZK).
+
+## Collaboration Space
+
+- Leave our collaboration space [Team Collaboration Space](https://www.samenwerkruimten.nl/teamsites/ai-validatie-team).
+
+## Shared email
+
+- Make sure to leave our  [shared email](../../About/contact.md) by sending an email to Secretariat of Digital Society.
```

Comment:
@anneschuth is the email of the Secretariat of Digital Society a publicly known mailbox? If yes, I would propose to make it easy and include the email address here. Or can we refer to an internal address book where people can find the email adres?